qemu/tests/cfg/qmp_command.cfg

```diff
@@ -147,8 +146,8 @@
             negative_test = yes
         - qmp_human-monitor-command:
             qmp_cmd = human-monitor-command command-line=info kvm
-            post_cmd = info kvm
-            cmd_result_check = m_in_q
+            cmd_result_check = contain
+            cmd_return_value = "kvm support: enabled"
```

Comment:
how about qemu 1.5.3 ? is it same value?

Reply:
yes, the same value, please check the following result
{"QMP": {"version": {"qemu": {"micro": 3, "minor": 5, "major": 1}, "package": " (qemu-kvm-1.5.3-141.el7_4.2)"}, "capabilities": []}}
{"execute":"qmp_capabilities"}
{"return": {}}
{"execute":"human-monitor-command","arguments":{"command-line":"info kvm"}}
{"return": "kvm support: enabled\r\n"}

Reply:
@CongLi  yes, the same value, please check the following executed result

{"QMP": {"version": {"qemu": {"micro": 3, "minor": 5, "major": 1}, "package": " (qemu-kvm-1.5.3-141.el7_4.2)"}, "capabilities": []}}
{"execute":"qmp_capabilities"}
{"return": {}}
{"execute":"human-monitor-command","arguments":{"command-line":"info kvm"}}
{"return": "kvm support: enabled\r\n"}